src/elements/element.point.js

```diff
@@ -79,7 +79,11 @@ module.exports = Element.extend({
 			return;
 		}
 
-		ctx.strokeStyle = vm.borderColor || defaultColor;
+
+		ctx.strokeStyle = helpers.valueOrDefault(vm.borderWidth, defaults.global.elements.point.borderWidth) === 0 ?
```

Comment:
I would only compute the line width once which improves performance and minification